examples/7_get_started_with_real_robot.md

```diff
@@ -0,0 +1,893 @@
+# Robots in the real-world
+
+This tutorial explains how to get started with real robots and train a neural network to control them autonomously.
+
+It covers how to:
+1. order and assemble your robot,
+2. connect your robot, configure and calibrate it,
+3. record your dataset and visualize it,
+4. train a policy on your data and make sure it's ready for evaluation,
+5. evaluate your policy and visualize the result afterwards.
+
+Following these steps, you will reproduce behaviors like picking a lego block and placing it in a bin with a relatively high success rate like in this video: https://x.com/RemiCadene/status/1814680760592572934
+
+While this tutorial is general and easily extendable to any type of robots by changing a configuration, it is based on the [Koch v1.1](https://github.com/jess-moss/koch-v1-1) affordable robot. Koch v1.1 is composed of a leader arm and a follower arm with 6 motors each. In addition, various cameras can record the scene and serve as visual sensors for the robot.
+
+During data collection, you will control the follower arm by moving the leader arm. This is called "teleoperation". It is used to collect robot trajectories.
+Then, you will train a neural network to imitate these trajectories. Finally, you will deploy your neural network to autonomously control your robot.
+
+Note: If you have issue at any step of the tutorial, ask for help on [Discord](https://discord.com/invite/s3KuuzsPFb).
+
+## 1. Order and Assemble your Koch v1.1
+
+Follow the sourcing and assembling instructions on the [Koch v1.1 github page](https://github.com/jess-moss/koch-v1-1) to setup the follower and leader arms shown in this picture.
+
+<div style="text-align:center;">
+  <img src="../media/tutorial/koch_v1_1_leader_follower.webp?raw=true" alt="Koch v1.1 leader and follower arms" title="Koch v1.1 leader and follower arms" width="50%">
+</div>
+
+See the [video tutorial of the assembly](https://youtu.be/5mdxvMlxoos).
+
+## 2. Configure motors, Calibrate arms, Teleoperate your Koch v1.1
+
+Connect the leader arm (the smaller one) with the 5V alimentation and the follower arm with the 12V alimentation. Then connect both arms to your computer with USB.
+
+**Pro tip**: In the next sections, you will understand how our code work by playing with python codes that you will copy past in your terminal. If you are new to this tutorial, we highly recommend to go through this. Next time, when you are more familiar, you can instead run the teleoperate script which will automatically:
+- detect a wrong configuration for the motors, and run the configuration procedure (explained after),
+- detect a missing calibration and start the calibration procedure (explained after),
+- connect the robot and start teleoperation (explained after).
+```bash
+python lerobot/scripts/control_robot.py teleoperate \
```

Comment:
Might be helpful to mention needing to run `pip install ".[koch]"` so the dynamixel-sdk is installed, and any other extras that may be needed for the tutorial.